json/helper.go

```diff
@@ -38,3 +38,24 @@ func Unmarshal(data []byte, i interface{}) error {
 
 	return nil
 }
+
+// Takes a value containing JSON string and passes it through
+// the JSON parser to normalize it, returns either a parsing
+// error or normalized JSON string.
+func NormalizeJsonString(jsonString interface{}) (string, error) {
+	var j interface{}
+
+	if jsonString == nil || jsonString.(string) == "" {
+		return "", nil
+	}
+
+	s := jsonString.(string)
```

Comment:
This could be simplified a bit:

```suggestion
	s, ok := jsonString.(type)
	if !ok || s == "" {
	  return "", nil
	}
```